test/JDBC/input/sys-server_principals-vu-verify.mix

```diff
@@ -15,6 +15,11 @@ FROM sys.server_principals
 WHERE name =  'sysadmin';
 GO
 
+SELECT name, type, type_desc, default_database_name, default_language_name, credential_id, owning_principal_id, is_fixed_role
+FROM sys.server_principals 
+WHERE name =  'public';
```

Comment:
Does this test case contain all the babelfish_authid_login_ext.type ?
Like the view sql in case when iterated 'R','U','S','Z' .